src/shared/components/case-editor/services/page-validation.service.ts

```diff
@@ -27,9 +27,9 @@ export class PageValidationService {
     return !(this.checkMandatoryField(caseField, theControl));
   }
 
-  private isHidden(caseField: CaseField, editForm: FormGroup) {
+  public isHidden(caseField: CaseField, editForm: FormGroup) {
```

Comment:
Should have a return type of `: boolean`.

Reply:
updated